### ISSUE TITLE
Xenoblade Chronicles 2 panicking on boot w/DLC & Update installed

### ISSUE THREAD

**Issue by kirbysuperstar** [yuzu-emu/yuzu#7484]
0100E95004038000 - Yuzu mainline 832

[yuzu_log.txt](https://github.com/yuzu-emu/yuzu/files/7630756/yuzu_log.txt)

The game has no problem booting without the update (which I believe means it doesn't really look at the DLC), or with the DLC off but update on (can't load the save because it was made with DLC on my Switch). I made sure my dump is valid and correct, tried OpenGL and Vulkan (didn't think this would matter but sometimes you never know) and all CPU accuracy levels, but it never gets past the loading bar after loading the shaders and never actually 'crashes', just sends a panic and hangs there.

**Comment by Morph1984** [comment]
Do you have all the (updated) DLC installed?

**Comment by gidoly** [comment]
you need to install all 6 dlc file to make game load properly

**Comment by kirbysuperstar** [comment]
Yes, Helpful Items, My Nintendo Bonus Items, New Challenge Mode, New Quests, New Rare Blade, Japanese Voices as well as Torna are all installed.

**Comment by gidoly** [comment]
did you installed torna the golden country?

**Comment by kirbysuperstar** [comment]
Yep, Torna is also installed.

**Comment by gidoly** [comment]
Last possibility i can think is title keys and prod keys are not updated for updates, dlc. I just tested on my system i can load all stuff without issue

**Comment by kirbysuperstar** [comment]
Turns out I needed to copy over my firmware files. Weird, but hey. The more you know. Thanks anyway!

**Comment by Morph1984** [comment]
I'll be reopening this issue as it's still valid.

**Comment by Crono141** [comment]
This issue is still occurring, using SteamDeck and AppImage yuzu 1162.  Here's my log.  I have 3 launches in this log.  First launch is game with both DLC (1 thru 6) and V2.1.0 enabled. This fails to boot with a traceback.

The second launch is with the DLC disabled and the game boots.

The third launch is with the update disabled.  The game boots but DLC is inaccessible, probably because the games needs an update to know the DLC exists at all.  The rom dump is from launch so as far as I know its V1.0 (no Version number on title screen).

[yuzu_log.txt](https://github.com/yuzu-emu/yuzu/files/9543681/yuzu_log.txt)

**Comment by FearlessTobi** [comment]
Is this still an issue?
